grafana/dashboards/drive-stats.json

```diff
@@ -807,7 +814,7 @@
           "group": [],
           "metricColumn": "none",
           "rawQuery": true,
-          "rawSql": "SELECT max(speed) AS speed_[[length_unit]]h\nFROM positions\nWHERE car_id = $car_id AND $__timeFilter(date);",
+          "rawSql": "SELECT convert_km(max(speed_max), '$length_unit') AS speed_${length_unit}h\nFROM drives\nWHERE car_id = $car_id and $__timeFilter(start_date);",
```

Comment:
@JakobLichterfeld

I've included the changes from #4323 (first commit) - I haven't found a way to really improve speed histogram without making use of advanced features of other DBMS (continuous aggregates...)